man/setcolorder.Rd

```diff
@@ -13,7 +13,7 @@ setcolorder(x, neworder)
 }
```

Comment:
`\usage{}` section will have to be updated as well

Reply:
must, otherwise R CMD check warns, best to run check before submitting PR as issues like this can be easily spotted.